Description: Patch to fix bug in plugin exim_mailstats, replace the wrong
 filehandle close through tail_close(). Fixed upstream in munin version 1.3.4,
 revision 1523
Origin: backport, http://munin.projects.linpro.no/changeset/1523
Bug-Debian: http://bugs.debian.org/486868
Index: munin-1.2.6/node/node.d/exim_mailstats.in
===================================================================
--- munin-1.2.6.orig/node/node.d/exim_mailstats.in	2008-08-14 06:06:50.000000000 +0200
+++ munin-1.2.6/node/node.d/exim_mailstats.in	2008-08-14 06:11:21.000000000 +0200
@@ -211,6 +211,8 @@
 
     while ($line = <$LOGFILE>) {
 	chomp ($line);
+	# delete the pid out if there is one (Debian #440622)
+	$line =~ s/^(.{19} )\[\d+\] /$1/; 
 
 	if (substr ($line, 37,2 ) eq '<=') {
 	    $received++;
@@ -220,7 +222,7 @@
             $rejected++;
         }
     }
-    close($LOGFILE) or die "Could not close $fname after reading: $!";
+    return tail_close($LOGFILE);
 }
 
 # vim:syntax=perl
